src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1468,6 +1476,91 @@ private function registerPropertyInfoConfiguration(array $config, ContainerBuild
         }
     }
 
+    /**
+     * Loads lock configuration.
+     *
+     * @param array            $config
+     * @param ContainerBuilder $container
+     * @param XmlFileLoader    $loader
+     */
```

Comment:
still, we should not add new ones. We have a lot of e.g. `Constructor.` docblock comments in the codebase, not worth removing them but we won't do the same for new constructors.